get-next-uid-gid.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+import sys
+import json
+import ldap
+import logging
+import argparse
+import rc_util
+from os import popen
+from rc_rmq import RCRMQ
+
+task = 'create_account'
+
+# Instantiate rabbitmq object
+rc_rmq = RCRMQ({'exchange': 'RegUsr', 'exchange_type': 'topic'})
+
+args = rc_util.get_args()
+
+# Logger
+logger = rc_util.get_logger()
+
+#Account creation 
+def create_account(msg):
+
+    logger.info(f'Account creation request received: {msg}')
+    username = msg['username']
+    uid = msg['uid']
+    email = msg['email']
+    fullname = msg['fullname']
+    success = False
+
+    try:
+        # Bright command to create user
+        cmd = '/cm/local/apps/cmd/bin/cmsh -c '
+        cmd += f'"user; add {username}; set userid {uid}; set email {email}; set commonname \\"{fullname}\\"; '
```

Comment:
`set userid` above doesn't work in brightCM (tested with version 9.0) when creating a user account. It should be changed to `set id` instead. Whether this issue is pertaining to the Bright version is to be seen.